shoop_tests/utils/test_enum.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from shoop.core.models.products import ProductMode
+from shoop.utils.enums import map_enum
+
+
+def test_map_enum():
+    assert map_enum(ProductMode, 0) is ProductMode.NORMAL
```

Comment:
That should be another function IMHO, since it's entirely different use case and having the possibility map from int here allows template developers to write code with magic numbers and we shouldn't make that easy.